Makefile

```diff
@@ -59,17 +59,17 @@ dv_review_aks: ## make dv_review_aks deploy APP_NAME=2222 CLUSTER=cluster1 USE_D
 	$(eval export TF_VAR_app_name=$(APP_NAME))
 	echo https://$(SERVICE_NAME)-review-$(APP_NAME).$(CLUSTER).development.teacherservices.cloud will be created in AKS
 
-qa_aks:
+qa:
 	$(eval include global_config/qa_aks.sh)
 	$(eval backup_storage_secret_name=PUBLISH-STORAGE-ACCOUNT-CONNECTION-STRING-DEVELOPMENT)
 
-staging_aks:
+staging:
 	$(eval include global_config/staging_aks.sh)
 
-sandbox_aks:
+sandbox:
 	$(eval include global_config/sandbox_aks.sh)
 
-production_aks:
+production:
```

Comment:
Hmm, I know this is out of scope for this card but do you know if we can swap these around? I feel like we should be consistent.

<img width="384" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/e583b4b7-c651-4c90-a0b1-92e6fa8fe477">

Reply:
Humm the problem is to make the same as a Apply but Apply doesn't have the production_aks environment.

I am strongly thinking that it might be the best in Publish to have both. make production ... and make production_aks ...

Wdyt?

Reply:
Why would we want both `make production` and `make production_aks` in Publish? Or I have I misunderstood?

Reply:
Sorry I answer because I understood another thing yesterday.